frontend/src/bundles/studio/mocks/mock.helper.ts

```diff
@@ -0,0 +1,79 @@
+import { minutesToMilliseconds } from 'date-fns';
+import {
+    type ItemDefinition,
+    type Range,
+    type RowDefinition,
+    type Span,
+} from 'dnd-timeline';
+
+const generateMockRows = (count: number): RowDefinition[] => {
+    return Array.from({ length: count })
+        .fill(0)
+        .map((_, index): RowDefinition => {
+            const disabled = false;
+
+            let id = `${index + 1}`;
+            if (disabled) {
+                id += ' (disabled)';
+            }
```

Comment:
if disable is always false then we don't need this?

Reply:
not fixed

Reply:
Fixed